protocols/virtualdefund/virtualdefund.go

```diff
@@ -203,12 +200,23 @@ func (o *Objective) finalState() state.State {
 	return state.StateFromFixedAndVariablePart(o.VFixed, vp)
 }
 
+// Returns the proposal with the largest amount
+// TODO: Should go
```

Comment:
Remove/Rename this

Reply:
Done in [d0f580e](https://github.com/statechannels/go-nitro/pull/826/commits/d0f580ed2460f7052aeda45bd8c03799ef8690de)